confluent_kafka/src/Consumer.c

```diff
@@ -508,6 +589,22 @@ static PyMethodDef Consumer_methods[] = {
 	  "  :raises: KafkaException\n"
 	  "\n"
 	},
+	{ "get_watermark_offsets", (PyCFunction)Consumer_get_watermark_offsets,
```

Comment:
Oh, for consistency I even just meant consistency across all the clients we're building based on librdkafka, but agreed that there could be others.

I'm fine with exposing it, just wanted to make sure we're thinking about compatibility and designing good, user friendly APIs along the way.